community/mit-scheme/depends

```diff
@@ -0,0 +1,3 @@
+gcc
+make
```

Comment:
These two lines can be removed. There's no use in listing `make`/`gcc` in depends as everything would depend on them

Reply:
4c943d39ec145208043885d328f58a57b989505f